specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/delegatedSubnets.json

```diff
@@ -0,0 +1,515 @@
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "An identifier that represents the resource."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The type of resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ResourceUpdateParameters": {
+      "description": "Parameters for updating a resource.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      }
+    },
+    "DelegatedSubnets": {
+      "description": "An array of DelegatedSubnet resources.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DelegatedSubnet"
+          },
+          "description": "An array of DelegatedSubnet resources."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DelegatedSubnet": {
+      "description": "Represents an instance of a orchestrator.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "readOnly": true,
+          "$ref": "#/definitions/DelegatedSubnetProperties",
+          "description": "Properties of the provision operation request."
+        }
+      }
+    },
+    "DelegatedSubnetProperties": {
+      "description": "Properties of delegated subnet",
+      "type": "object",
+      "properties": {
+        "resourceGuid": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource guid."
+        },
+        "state": {
+          "type": "string",
+          "enum": [
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Provisioning"
+          ],
+          "x-ms-enum": {
+            "name": "delegated subnet resource state",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "The current state of dnc delegated subnet resource."
+        },
+        "subnetDetails": {
+          "description": "orchestrator details",
+          "$ref": "#/definitions/subnetDetails"
+        },
+        "controllerDetails": {
```

Comment:
If subnets belong to a controller why not have them as children of the controller?  Can they switch controllers during their lifetime?  Do they have to be the same region as the subnet/controller?  Who validates that?